### PR TITLE
edit user data binding  + merge fixes

### PR DESCRIPTION
**Pull request by ckiss**

### CODE REVIEW

**Comment by ckiss** [comment]
@mcdonnelldean this is ready to review now. Edit user should work as expected

File: client/containers/login.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export const Login = React.createClass({
 
   handleSubmit (event) {
     event.preventDefault()
+    debugger
```
Comment (mcdonnelldean) [inline client/containers/login.js]: remove

File: client/containers/users.js [hunk 2]
```diff
@@ -16,8 +18,8 @@ export const Users = React.createClass({
     this.props.dispatch(pushPath('user/add'))
   },
 
-  handleEdit (id) {
-    this.props.dispatch(pushPath(`user/${id}/edit`))
```
Comment (mcdonnelldean) [inline client/containers/users.js:20]: Why was this changed?

Comment (ckiss) [inline client/containers/users.js:20]: cause using this I wouldn't have been able to get the user I want to edit in getInitialState (check this https://github.com/nearform/concorda/pull/54/files#diff-6acd45333fc718ab57b294b3a3693b2cR12)
